my-app/src/components/ContentList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+const ContentList = ({content}) => {
+    return (
+        <div className="cards_box">
+     { content && content.map(content=>(  
```

Comment:
have you forgotten to add your `key` to the list by any chance?

Reply:
oopsie :sweat_smile: